### ISSUE TITLE
RG_NAMES are not applying default values

### ISSUE THREAD

**Issue by jmservera** [martinabrle/aks-java-demo#3]
According to the documentation: 
https://github.com/martinabrle/aks-java-demo/blob/65da0fb47066f71693d4fbfd6b02cce84fd9fc33/docs/aks-github-actions.md?plain=1#L19C1-L19C35

The values:
```
CONTAINER_REGISTRY_RESOURCE_GROUP (if omitted, the value of AKS_RESOURCE_GROUP will be used)
CONTAINER_REGISTRY_SUBSCRIPTION_ID (if omitted, the value of AKS_SUBSCRIPTION_ID will be used)
LOG_ANALYTICS_WRKSPC_RESOURCE_GROUP (if omitted, the value of AKS_RESOURCE_GROUP will be used)
LOG_ANALYTICS_WRKSPC_SUBSCRIPTION_ID (if omitted, the value of AKS_SUBSCRIPTION_ID will be used)
PGSQL_RESOURCE_GROUP (if omitted, the value of AKS_RESOURCE_GROUP will be used)
PGSQL_SUBSCRIPTION_ID (if omitted, the value of AKS_SUBSCRIPTION_ID will be used)
```
Should fall back to default values, but neither the action script nor the bicep file do that resulting in a failure:

```log
Run az deployment sub create \
ERROR: {"code": "MultipleErrorsOccurred", "message": "Multiple error occurred: BadRequest,BadRequest,BadRequest. Please see details."}

Inner Errors: 
{"code": "InvalidTemplate", "target": "/subscriptions/***/providers/Microsoft.Resources/deployments/log-analytics-rg", "message": "Deployment template validation failed: 'The template resource '' of type 'Microsoft.Resources/resourceGroups' at line '1' and column '787' is not valid. The name property cannot be null or empty. Please see https://aka.ms/arm-syntax-resources for usage details.'.", "additionalInfo": [{"type": "TemplateViolation", "info": {"lineNumber": 1, "linePosition": 787, "path": "properties.template.resources[0]"}}]}

Inner Errors: 
{"code": "InvalidTemplate", "target": "/subscriptions/***/providers/Microsoft.Resources/deployments/pgsql-rg", "message": "Deployment template validation failed: 'The template resource '' of type 'Microsoft.Resources/resourceGroups' at line '1' and column '771' is not valid. The name property cannot be null or empty. Please see https://aka.ms/arm-syntax-resources for usage details.'.", "additionalInfo": [{"type": "TemplateViolation", "info": {"lineNumber": 1, "linePosition": 771, "path": "properties.template.resources[0]"}}]}

Inner Errors: 
{"code": "InvalidTemplate", "target": "/subscriptions/***/providers/Microsoft.Resources/deployments/container-registry-rg", "message": "Deployment template validation failed: 'The template resource '' of type 'Microsoft.Resources/resourceGroups' at line '1' and column '797' is not valid. The name property cannot be null or empty. Please see https://aka.ms/arm-syntax-resources for usage details.'.", "additionalInfo": [{"type": "TemplateViolation", "info": {"lineNumber": 1, "linePosition": 797, "path": "properties.template.resources[0]"}}]}
Error: Process completed with exit code 1.
```

**Comment by martinabrle** [comment]
possibly fixed by commit nr 5c687fb

**Comment by martinabrle** [comment]
This is fixed now
